### PR TITLE
Adds sushi to foods and suggest role tutorial

### PR DESCRIPTION
**Pull request by hflyn**
makisushi

### CODE REVIEW

File: NextTime.yaml [hunk 1]
```diff
@@ -11,3 +11,4 @@ suggestions:
   - Building codebase (B script)
   - CMakeLists
   - Making a module
+  - Making a role
```
Comment (Claegtun) [inline NextTime.yaml:25]: Writing a comment

**Comment by TasneemMuneera** [comment]
I love sushi

**Comment by ysims** [comment]
Looks like you need to fix up the merge conflicts then should be good :)

File: NextTime.yaml (hunk 1 above)
Comment (ysims) [inline NextTime.yaml:25]: If you're interested in learning more about roles, there is [this NUbook page](https://nubook.nubots.net/system/foundations/overview#roles). I'm also planning to extend the Getting Started page to include a tutorial that will include roles :smile:, and we could do a workshop on it. 
Nice suggestion!

**Review by ysims — APPROVED** [review]
LGTM

**Pull request closed without merging**